modules/go-health-monitoring.adoc

```diff
@@ -20,4 +20,6 @@ The {gitops-title} Operator will install the GitOps backend service in the `open
 . If the application is `out of sync` or `degraded` applications the respecting icons are displayed under the *Resources*. Hover over the icons to see the health status and the sync status. The icons are:
 
 .. For `degraded`, the broken heart icon is displayed.
-.. For `out of sync`, the yellow yield icon is displayed.
\ No newline at end of file
+.. For `out of sync`, the yellow yield icon is displayed.
```

Comment:
For `out of sync` status, the yellow yield icon is displayed.

NOTE: You might consider using inline images. See https://docs.asciidoctor.org/asciidoc/latest/macros/images/#inline-image-macro. However, I'm not sure about the best practice for doc maintenance and rendering; get a second pinion from a more experienced reviewer.

Reply:
See my comment above. Consult the ISG for specific guidance about how to refer to and use icons.

Reply:
Not using icons since to maintain consistency. Till now we have rarely used icons